### PR TITLE
Added new route to generate unique token

### LINKED ISSUE #940: Feature request: GET api call to send unique token to my site
We're implementing Sign in with QR code feature in mobile app.

For more information you can check https://github.com/Real-Dev-Squad/mobile-app/issues/146 this out

Feel free to tag me  in mobile-app channel, if you want any further clarifications.

### PR DESCRIPTION
**Pull request by Satyam005**
# Fixes Issue

Closes #940 

## Changes proposed
GET api call to send unique token to my site

## What did you do ?
- Used uuid package to generate a 16-byte unique value
- Created a endpoint **_generateUniqueToken_** to generate a unique token

## Screenshots
_controllers/generateUniqueToken.js_

image

_routes/generateUniqueToken.js_

image

**_Response_** 
image

### CODE REVIEW

File: controllers/generateUniqueToken.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const uuid = require("uuid");
+
+/**
+ * Route used to generate a unique token
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const generateToken = (req, res) => {
+  const token = uuid.v4();
```
Comment (shreya-mishra) [inline controllers/generateUniqueToken.js:10]: do we have the uuid package already installed?

**Review by shreya-mishra — COMMENTED** [review]
can you also attach the response as a screenshot, please?

File: controllers/generateUniqueToken.js (hunk 1 above)
Comment (Satyam005) [reply controllers/generateUniqueToken.js:10]: Response screenshot added @shreya-mishra

Comment (Satyam005) [reply controllers/generateUniqueToken.js:10]: > do we have the uuid package already installed?

No @shreya-mishra , it was not there, i have added it now.

**Comment by shreya-mishra** [comment]
closing since, requirement has changed

**Pull request closed without merging**